indra/resources/grounding/covid_grounding.csv

```diff
@@ -78,4 +78,4 @@ CpG,NCIT,C67101
 ACh,CHEBI,CHEBI:15355
 age,MESH,D000375
 Age,MESH,D000375
-CatL,HGNC,CTSL
```

Comment:
maybe would be a good idea to have a unit test that validates the entries in these CSVs as having valid prefix/identifier pairs -> I just noticed that HGNC:CTSL is definitly not a valid HGNC identifier

Reply:
I agree, although it's possible the tests are already in place and will soon fail. Also you've just reminded me of something important. For historical reasons, the HGNC entries in our grounding maps use HGNC symbols instead of HGNC IDs. I actually had forgotten about his when adding all of the new entries and used the IDs instead. The script I wrote to automatically add entries from a curation sheet does validate the pairs, but the validation expects IDs instead of symbols. I think having these seemingly arbitrary exceptions to standard rules makes the need for testing even more important.

Reply:
I don't plan to work on this further over the weekend. On Monday, I will change the grounding map entries I've added with HGNC IDs to use HGNC symbols, change the update script to accept either ID or symbol in a curation spreadsheet, validate appropriately, and add entries to the grounding map files always using symbols. We can all discuss the testing situation during the standup.

Reply:
In this file the correct convention - for better or worse - is to use HGNC symbols. So this is okay as is.